sdk/formrecognizer/azure-ai-formrecognizer/README.md

```diff
@@ -126,17 +137,16 @@ with the Azure SDK, please install the `azure-identity` package:
 
 ```pip install azure-identity```
 
-You will also need to [register a new AAD application and grant access][register_aad_app] to
-Form Recognizer by assigning the `"Cognitive Services User"` role to your service principal.
+You will also need to [register a new AAD application][register_aad_app] and [grant access][aad_grant_access] to Form Recognizer by assigning the `"Cognitive Services User"` role to your service principal.
```

Comment:
the grant access link is broken since I'm assuming we dont have that link in the README.